sslh-fork.c

```diff
@@ -158,6 +165,17 @@ void main_loop(int listen_sockets[], int num_addr_listen)
         case 0:
             /* Listening process just accepts a connection, forks, and goes
              * back to listening */
+
+#ifdef LIBBSD
```

Comment:
I'd rather have this moved to a `set_listen_procname` or something like that (to keep the `main_loop` tidy), and make the preprocessor conditionals in that function rather than here, like we do in `common.c` for LIBCAP, LIBWRAP and so on. In general, it's good practice to keep pre-processor down to a minimum to avoid what we have in `bind_peer()` which is very confusing.

Reply:
I've addressed this comment in a follow-up commit. I intend to squash these together before merging, but this should make your review easier.